CHANGELOG.md

```diff
@@ -6,12 +6,24 @@ The Azure MCP Server updates automatically by default whenever a new release com
 
 ### Features Added
 
+- Introduced `BaseAzureResourceService` class that uses Azure Resource Graph queries for improved performance and reduced API calls when retrieving Azure SQL resources. This provides faster and more efficient resource discovery compared to direct ARM API calls.
```

Comment:
You need to link to the PR.  See other changelog entries for example

Reply:
updated